src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/CompileFails.cs

```diff
@@ -209,5 +209,18 @@ public async Task ValidateDisableRuntimeMarshallingForBlittabilityCheckFromAssem
 
             TestUtils.AssertPostSourceGeneratorCompilation(newComp);
         }
+
+        [Fact]
+        public async Task ValidateRequireAllowUnsafeBlocksDiagnostic()
```

Comment:
Is there a test that validates the diagnostic isn't triggered if LibraryImportAttribute isn't used, both with and without AllowUnsafeBlocks?

Reply:
I didn't add a test for that because since the product has multiple projects like that, it would be found with a product build. I can add one if you think there is a value though.

Reply:
I think it'd be worthwhile to add one.  Thanks.